pubsublite/spark-connector/README.md

```diff
@@ -0,0 +1,184 @@
+# Using Spark SQL Streaming with Pub/Sub Lite
+
+The samples in this directory show how to read messages from and write messages to Pub/Sub Lite from an [Apache Spark] cluster using the [Pub/Sub Lite Spark Connector].
+
+Visit this Maven [link](https://search.maven.org/search?q=g:com.google.cloud%20a:pubsublite-spark-sql-streaming) to download the connector's uber jar. The uber jar has a "with-dependencies" suffix. You will need to include it on the driver and executor classpaths when submitting a Spark job, typically in the `--jars` flag. Alternatively, you can get this JAR from this [public Cloud Storage location](gs://spark-lib/pubsublite/pubsublite-spark-sql-streaming-LATEST-with-dependencies.jar).
+
+## Before you begin
+
+1. Install the [Cloud SDK].
+   > *Note:* This is not required in [Cloud Shell]
+   > because Cloud Shell has the Cloud SDK pre-installed.
+
+1. Create a new Google Cloud project via the
```

Comment:
curious why this is `1.`?

Reply:
That's just the markdown format, either a series of numbers with `1. 1. 1.` or `1. 2. 3.` are both accepted, and both give the same result. See https://github.com/GoogleCloudPlatform/python-docs-samples/tree/pubsublite/pubsublite/spark-connector for preview of this file.

Reply:
SG!